dali/pipeline/operators/decoder/image_decoder.cc

```diff
@@ -49,6 +49,11 @@ is encountered and internal buffer needs to be reallocated to decode it.)code",
 `batch_size*prefetch_queue_depth` during the construction and suballocate them
 in runtime. Ignored when `split_stages` is false.)code",
       false)
+  .AddOptionalArg("use_fast_dct",
+      R"code(Enables fast IDCT in libjpeg-turbo decompressor when the library is used (CPU JPEG decoding).
```

Comment:
```suggestion
      R"code(Enables fast IDCT in CPU based decompressor when the library is used (CPU JPEG decoding).
```
Add note that it applies when the GPU implementation fallback to CPU as well.